proxy/README.md

```
@@ -0,0 +1,55 @@
+```bash
+sudo apt update && sudo apt install openssl
+```
+
+## Step 1: Generate Certificates
+
+Go into `proxy/alpha-sepolia-certs` directory.
+
+### Commands
+
+1. **Generate CA Private Key**:
+    ```bash
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)